modules/installation-and-upgrade/pages/server-major-upgrade-uyuni.adoc

```diff
@@ -21,6 +21,11 @@ For more information about backing up {productname}, see xref:administration:bac
 If you are running {productname} Server on a virtual machine, we recommend that you create a snapshot before you start.
 ====
 
+[IMPORTANT]
+====
+Before the upgrade, you need to ensure that the storage requirements are still valid. Refer to uyuni-install-requirements.adoc. Migration can fill root partition if there is not enough space available due to the service pack migration and download of new rpms. It is the same for the /var/lib/pgsql, when upgrading postgresql, it takes a copy of the old database, be sure to have at least enough space available to cope with a copy of the database.
```

Comment:
Some syntax fixes, I didn't notice before, sorry about it.
```suggestion
Before the upgrade, you need to ensure that the storage requirements are still valid. Refer to xref:uyuni-install-requirements.adoc[]. Migration can fill root partition if there is not enough space available due to the service pack migration and download of new RPMs. It is the same for the `/var/lib/pgsql`, when upgrading PostgreSQL, it takes a copy of the old database, be sure to have at least enough space available to cope with a copy of the database.
```
Fix for the link, highlight the path, adjust some naming :-)